Source/Core/Common/Matrix.cpp

```diff
@@ -360,4 +365,17 @@ void Matrix44::Multiply(const Matrix44& a, const Vec4& vec, Vec4* result)
   result->data = MatrixMultiply<4, 4, 1>(a.data, vec.data);
 }
 
+float Matrix44::Determinant() const
+{
+  const auto m = data;
```

Comment:
```suggestion
  const auto& m = data;
```

Perhaps, so it's not making a copy of the underlying array.